Frontend/src/Components/Graph/GraphSelectModal.js

```diff
@@ -85,37 +91,45 @@ function GraphSelectModal(props) {
   const [historyLength, setHistoryLength] = useState(initialHistoryLength);
 
   return (
-    <Modal isOpen={isOpen} onClose={onClose} size="xl">
+    <Modal isOpen={isOpen} onClose={onClose} size="3xl">
       <ModalOverlay />
       <ModalContent>
         <ModalHeader>Select Datasets</ModalHeader>
         <ModalCloseButton />
-
         <ModalBody>
           <VStack align="stretch">
-            {GraphData.categories.map((category) => (
+            <hr />
+            {GraphData.Output.map((category) => (
               <VStack align="stretch" key={category.category}>
-                <Heading size="md">{category.category}</Heading>
-                <SimpleGrid columns={2}>
-                  {category.values.map((value) => (
-                    <Checkbox
-                      defaultChecked={dataKeys.includes(value.key)}
-                      onInput={(e) => {
-                        // console.log(value.name, e.target.checked);
-                        if (e.target.checked) {
-                          // checked => add to checked datasets
-                          changeDataKeys({ action: "add", key: value.key });
-                        } else {
-                          // unchecked => remove from checked datasets
-                          changeDataKeys({ action: "remove", key: value.key });
-                        }
-                      }}
-                      key={value.key}
-                    >
-                      {value.name}
-                    </Checkbox>
+                <Expandable label={category.category} contentBg={contentBg} size="md">
```

Comment:
```suggestion
                <Expandable label={category.category} contentBg={contentBg} size="lg">
```
Thoughts on changing the button size to `lg`? `md` looks to be about the same size as the subcategory headings. Not a necessary change, but I'd like to know your thoughts.

Reply:
fixed